.potato.yml

```
@@ -0,0 +1,26 @@
+# This is the configuration file with settings for Potato. Potato is an internal Yandex technology.
+# Potato allows us to automate routine actions and sync internal [Yandex.Tracker](https://yandex.com/tracker/) and Github.
+# Now for all PRs where documentation is needed, just put a label 'need-docs' and we will plan PR to documentation sprints.
+
+# The project name.
+name: clickhouse
+# Object handlers defines which handlers we use.
+handlers:
+  # The handler for creating an Yandex.Tracker issue.
+  - name: issue-create
+  params:
+    triggers:
+      # The trigger for creating the Yandex.Tracker issue. When the specified event occurs, it transfers PR data to Yandex.Tracker.
+      github:pullRequest:labeled:
+        data:
+          # The Yandex.Tracker queue to create the issue in. Each issue in Tracker belongs to one of the project queues.
+          queue: CLICKHOUSEDOCS
+          # The issue title.
+          summary: [Potato] Задача для Pull Request {{pullRequest.number}}
```

There's a syntax error, it can be fixed with:

```suggestion
          summary: >
            [Potato] Задача для Pull Request {{pullRequest.number}}
```

... or with quotes:

```suggestion
          summary: '[Potato] Задача для Pull Request {{pullRequest.number}}'
```

P.S. you can use any online-tool to validate YAML format, e.g. https://www.json2yaml.com